src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return color.toString();
```

Comment:
Use name() for getting String representation of enum constants. Instead of using `color.toString()`, you should use `color.name()` to ensure the correct string representation is returned, especially if `toString()` is overridden in the enum.

Reply:
fix comment